easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.9.0-GCC-8.3.0.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'CMakeMakeCp'
+
+name = 'RAxML-NG'
+version = '0.9.0'
+
+local_terrap_commit = '6dcb061ceb6ed41a7ebe1a8af1f3354e0126125c'
+local_pllmod_commit = '840a19525ed133ad243afb23124911b849d4685d'
+local_libpll_commit = '6874327c9ca9544e27b426c0e7205c8b6f4ed297'
+
+homepage = "https://github.com/amkozlov/raxml-ng"
+description = """RAxML-NG is a phylogenetic tree inference tool which
+uses maximum-likelihood (ML) optimality criterion. Its search heuristic
+ is based on iteratively performing a series of Subtree Pruning and Regrafting
+ (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/amkozlov/raxml-ng/archive/'],
+        'filename': '%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/amkozlov/terraphast-one/archive/'],
+        'download_filename': '%s.zip' % local_terrap_commit,
+        'filename': 'terraphist-one.zip',
+    },
+    {
+        'source_urls': ['https://github.com/ddarriba/pll-modules/archive/'],
+        'download_filename': '%s.zip' % local_pllmod_commit,
+        'filename': 'pll-modules.zip',
+    },
+    {
+        'source_urls': ['https://github.com/xflouris/libpll-2/archive/'],
+        'download_filename': '%s.zip' % local_libpll_commit,
+        'filename': 'libpll-2.zip',
+    },
+]
```

Comment:
This is just a minor suggestion, but we can achieve the same result of all the commands in `preconfigopts` with a tailored `extract_cmd` as follows
```suggestion
local_extract_lib = 'tar --strip-components=1 -C %%(builddir)s/raxml-ng-%%(version)s/libs/%s -xzf %%s'

sources = [
    {
        'source_urls': ['https://github.com/amkozlov/raxml-ng/archive/'],
        'filename': '%(version)s.tar.gz',
    },
    {
        'source_urls': ['https://github.com/amkozlov/terraphast-one/archive/'],
        'download_filename': '%s.tar.gz' % local_terrap_commit,
        'filename': 'terraphast-one-%(version)s.tar.gz',
        'extract_cmd': local_extract_lib % 'terraphast',
    },
    {
        'source_urls': ['https://github.com/ddarriba/pll-modules/archive/'],
        'download_filename': '%s.tar.gz' % local_pllmod_commit,
        'filename': 'pll-modules-%(version)s.tar.gz',
        'extract_cmd': local_extract_lib % 'pll-modules',
    },
    {
        'source_urls': ['https://github.com/xflouris/libpll-2/archive/'],
        'download_filename': '%s.tar.gz' % local_libpll_commit,
        'filename': 'libpll-2-%(version)s.tar.gz',
        'extract_cmd': local_extract_lib % 'pll-modules/libs/libpll',
    },
]
```
Apart from a adding the additional `extract_cmd` in the three additional libs, I would also add a version string to the downloaded sources. Otherwise they could be overwritten by future updates of this easyconfig.